src/Nethermind/Nethermind.Consensus/Producers/TxPoolTxSource.cs

```diff
@@ -73,6 +74,19 @@ public IEnumerable<Transaction> GetTransactions(BlockHeader parent, long gasLimi
                 bool success = _txFilterPipeline.Execute(tx, parent);
                 if (success)
                 {
+                    if (tx.Type == TxType.Blob)
+                    {
+                        if ((blobsCounter + tx.BlobVersionedHashes.Length) > 4)
```

Comment:
this way we will avoid magic number `4` here

Reply:
Yes, it'd be cool to move it to a common constant, I will do it in validation PR